### PR TITLE
DP-1350 Set dataset source type in creation wizard

### PR DESCRIPTION
**Pull request by simenheg**

### CODE REVIEW

File: okdata/cli/commands/datasets/boilerplate/config.py [hunk 1]
```diff
@@ -127,6 +140,7 @@ def boilerplate_prompt(include_extra_metadata=True):
             "name": "pipeline",
             "message": "Prosessering",
             "choices": pipeline_choices,
+            "when": lambda x: x["sourceType"] == "file",
```
Comment (fredriv) [inline okdata/cli/commands/datasets/boilerplate/config.py:143]: Also for `event`?

Comment (fredriv) [reply okdata/cli/commands/datasets/boilerplate/config.py:143]: Requiers a different set of pipeline choices (e.g. event-copy, none)

File: okdata/cli/commands/datasets/wizards.py [hunk 2]
```diff
@@ -86,9 +87,20 @@ def start(self):
...
   okdata datasets cp FILE ds:{dataset_id}
 """
-        )
+            )
+        elif choices["sourceType"] == "event":
+            # TODO: Just create the event stream automatically too?
+            self.command.print(
+                f"""Done! You may go ahead and create an event stream:
+
+  okdata events create-stream {dataset_id}
```
Comment (fredriv) [inline okdata/cli/commands/datasets/wizards.py:102]: Add example of how to send events to it once the stream is created?

File: okdata/cli/commands/datasets/boilerplate/config.py (hunk 1 above)
Comment (simenheg) [reply okdata/cli/commands/datasets/boilerplate/config.py:143]: Sure!

However `event-transform` doesn't give any value out of the box, right? So maybe offer `event-copy` and `none` for now, and still require manual configuration for setting up transformations?

**Review by fredriv — APPROVED** [review]
LGTM :+1:

**Pull request merged**